jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/cpu/ContextSwitchRateHandler.java

```diff
@@ -0,0 +1,44 @@
+package io.opentelemetry.contrib.jfr.metrics.cpu;
+
+import io.opentelemetry.api.metrics.*;
+import io.opentelemetry.contrib.jfr.Constants;
+import io.opentelemetry.contrib.jfr.metrics.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+
+public class ContextSwitchRateHandler implements RecordedEventHandler {
+  public static final String EVENT_NAME = "jdk.ThreadContextSwitchRate";
+
+  private final Meter otelMeter;
+  private volatile double value = 0;
+
+  public ContextSwitchRateHandler(Meter otelMeter) {
+    this.otelMeter = otelMeter;
+  }
+
+  public ContextSwitchRateHandler init() {
+    otelMeter.upDownCounterBuilder("jfr.ThreadContextSwitchRate")
```

Comment:
Same guidance as CPU.

Now, if the context-switch-rate is actually PART of the cpu usage that's dedicated to thread-context-switching, then I'd suggest using `runtime.jvm.cpu.utilization` with an appropriate Attribute denoting `state=thread.context_switch`.

MIght be worth documenting the metrics we plan to expose in their own Markdown file, similar to what the collector does here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/metadata.yaml